packages/contracts-bedrock/scripts/deploy/UpgradeAnchorStateRegistry.s.sol

```diff
@@ -73,13 +73,13 @@ contract UpgradeAnchorStateRegistry is Script {
     ) internal {
         address anchorStateRegistryImpl = DeployUtils.create1({
             _name: "AnchorStateRegistry",
-            _args: abi.encode(_disputeGameFactoryProxy)
+            _args: DeployUtils.encodeConstructor(_disputeGameFactoryProxy)
```

Comment:
This will strips the first 4 bytes but there's no redundant bytes here.

Reply:
So what is your suggestion to fix this?

Reply:
Maybe like this: https://github.com/ethereum-optimism/optimism/blob/1add88a41b8cdc5c488a18976e3e4fa4b02e7da4/.semgrep/tests/sol-rules.t.sol#L65

Reply:
done